Create legacy maintenance branch for v0.11.1

jakirkham opened the pull request:
Refreshes the legacy `v0.11.1` branch using the latest `conda-smithy`. Needed for `airflow`. Re-renders with the latest `conda-smithy` and converts the build to `noarch: python`.

cc @sodre

<hr>

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

Fixes https://github.com/conda-forge/flask-feedstock/issues/16

<!--
Please add any other relevant info below:
-->

jakirkham committed:
MNT: Re-rendered with conda-smithy 3.1.5 and pinning 2018.05.22

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham committed:
Convert package to noarch Python

As this is a pure Python package with no conditional constraints, it is
a good candidate for converting to a `noarch: python` package. This
makes it easy to install under various Python environments and cuts down
on the associated maintenance burden. Given this, go ahead and convert
the recipe to `noarch: python`.

jakirkham committed:
MNT: Re-rendered with conda-smithy 3.1.5 and pinning 2018.05.22

Now that this is a `noarch: python` package, re-render to update the CIs
and other feedstock content accordingly.

jakirkham committed:
Use pip to build and install

As `pip` is now the preferred way to build and install Python packages
in conda-forge, switch over to using `pip`. This is especially important
now that this is a `noarch: python` package.

jakirkham committed:
Bump build number to 1

As the package is now `noarch: python` and built with `pip`, bump the
build number to generate a new package.

jakirkham commented:
Ok to merge?

marcelotrevisani commented:
LGTM !

Thanks @jakirkham  ! :)

jakirkham commented:
Thanks @marcelotrevisani :)
